tests/Microsoft.Bot.Connector.Tests/BaseTest.cs

```diff
@@ -38,6 +38,7 @@ public class BaseTest
 
         public BaseTest()
         {
+            ServiceClientTracing.IsEnabled = true;
```

Comment:
I don't understand what this is doing. Would you add a detailed comment?

Reply:
We are making sure service client tracing is enabled in unit tests to make them go through the [`_shouldTrace` condition](https://github.com/Microsoft/botbuilder-dotnet/blob/master/libraries/Microsoft.Bot.Connector/Conversations.cs#L273-L279).

```
// Tracing
bool _shouldTrace = ServiceClientTracing.IsEnabled;
string _invocationId = null;
if (_shouldTrace)
{
    _invocationId = ServiceClientTracing.NextInvocationId.ToString();
    Dictionary<string, object> tracingParameters = new Dictionary<string, object>();
    tracingParameters.Add("parameters", parameters);
    tracingParameters.Add("cancellationToken", cancellationToken);
    ServiceClientTracing.Enter(_invocationId, this, "CreateConversation", tracingParameters);
}
```

Reply:
We started investigating a better approach to test tracing features as exemplified in #1012. We'll hold off this PR and implement a similar approach.